tests/Backend/CommonPart1/ExportParamsTest.php

```diff
@@ -209,4 +210,27 @@ public function testTableExportAsyncPermissions(): void
         $file = $client->getFile($results['file']['id']);
         Client::parseCsv(file_get_contents($file['url']), false);
     }
+
+
+    public function testExportWithInternalTimestampColumn(): void
+    {
+        $this->allowTestForBackendsOnly([
+            self::BACKEND_SNOWFLAKE,
+        ]);
+        $downloadPath = $this->getExportFilePathForTest('languages.sliced.csv');
+        $filePath = __DIR__ . '/../../_data/languages.csv';
+        $importFile = new CsvFile($filePath);
+
+        $tableId = $this->_client->createTableAsync($this->getTestBucketId(), 'exportWithTimestamp', $importFile);
+
+        $exporter = new TableExporter($this->_client);
+        $exporter->exportTable($tableId, $downloadPath, [
+            'includeInternalTimestamp' => true,
+        ]);
+
+        // compare data
+        $this->assertFileExists($downloadPath);
+        $exportFile = new CsvFile($downloadPath);
+        $this->assertSame(['id','name','_timestamp'], $exportFile->getHeader());
```

Comment:
funguje to, premyslam ci to nebude trochu matuce ze ak ma niekto presne definovane columns cez parameter 
```
'columns' => ['id', 'name'],
```
tak mu tam pribudne este jeden stlpec, mozno by som to spomenul aspon v doc